source/FloydPink.Flickr.Downloadr.Logic/DownloadLogic.cs

```diff
@@ -77,6 +77,11 @@ public DownloadLogic(IOriginalTagsLogic originalTagsLogic)
         }
 
         var photoName = preferences.TitleAsFilename ? GetSafeFilename(photo.Title) : photo.Id;
+
+        if (preferences.FlickrOrder) {
+          photoName = curCount.ToString().PadLeft(8, '0');
```

Comment:
perhaps the same padding could be applied for line # 90 below? 
and reduce the number of digits to maybe 6...